docs/install_guide.adoc

```diff
@@ -0,0 +1,206 @@
+NAME                        HOST/PORT                                              PATH   SERVICES                    PORT   TERMINATION     WILDCARD
+metric-storage-prometheus   metric-storage-prometheus-openstack.apps-crc.testing          metric-storage-prometheus   web    edge/Redirect   None
+----
+
+Prometheus metrics will be viewable in a web browser at the `HOST/PORT` address returned above. ie:
+https://metric-storage-prometheus-openstack.apps-crc.testing.
+
+== Installing the Operator
+
+.Procedure
+
+. Now that you have a ready working environment, you can install the Watcher Operator.
+First, create a `watcher-operator.yaml` file:
++
+[source,yaml]
+----
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: watcher-operator-index
+  namespace: openstack-operators
+spec:
+  image: quay.io/openstack-k8s-operators/watcher-operator-index:latest
+  sourceType: grpc
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openstack
+  namespace: openstack-operators
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: watcher-operator
+  namespace: openstack-operators
+spec:
+  name: watcher-operator
+  channel: alpha
+  source: watcher-operator-index
+  sourceNamespace: openstack-operators
+----
++
+. With cluster-admin permissions in Openshift run:
++
+[,console]
+----
+$ oc apply -f watcher-operator.yaml
+catalogsource.operators.coreos.com/watcher-operator-index created
+operatorgroup.operators.coreos.com/openstack unchanged
+subscription.operators.coreos.com/watcher-operator created
+----
++
+. Check that the operator is installed:
++
+[,console]
+----
+$ oc get subscription.operators.coreos.com/watcher-operator -n openstack-operators
+NAME               PACKAGE            SOURCE                   CHANNEL
+watcher-operator   watcher-operator   watcher-operator-index   alpha
+
+$ oc get pod -l openstack.org/operator-name=watcher -n openstack-operators
+NAME                                                  READY   STATUS    RESTARTS   AGE
+watcher-operator-controller-manager-dd95db756-kslw9   2/2     Running   0          44s
+----
+
+== Deploying the Watcher Service
+
+Now, you will need to create a Watcher Custom Resource based on the `Watcher CRD`.
+
+.Procedure
+
+. Use the following commands to _view_ the `Watcher CRD` definition and specification schema:
++
+[,console]
+----
+$ oc describe crd watcher
+
+$ oc explain watcher.spec
+----
++
+. Add a `WatcherPassword` field to the `Secret` created as part of the ControlPlane deployment as explained in https://docs.redhat.com/en/documentation/red_hat_openstack_services_on_openshift/18.0/html/deploying_red_hat_openstack_services_on_openshift/assembly_preparing-RHOCP-for-RHOSO#proc_providing-secure-access-to-the-RHOSO-services_preparing[Providing secure access to the Red Hat OpenStack Services on OpenShift services]. Then `oc apply` the updated file and check the `WatcherPassword` is included.
```

Comment:
I suggest splitting this into two steps:

. Add a `WatcherPassword` field to the `Secret` created as part of the ControlPlane deployment as explained in https://docs.redhat.com/en/documentation/red_hat_openstack_services_on_openshift/18.0/html/deploying_red_hat_openstack_services_on_openshift/assembly_preparing-RHOCP-for-RHOSO#proc_providing-secure-access-to-the-RHOSO-services_preparing[Providing secure access to the Red Hat OpenStack Services on OpenShift services].

. Update the `Secret`, and verify that the `WatcherPassword` field is present: